Exception while importing docxml

CI process was reported as failing with the error

```
2018-11-21T16:44:13.3591020Z Updating: Microsoft.ML.Transforms.Normalizers.MinMaxDblAggregator
2018-11-21T16:44:13.4260041Z ##[error]mdoc: System.Exception: Unable to translate generic parameter T; genDeclType=Microsoft.ML.Transforms.Normalizers.MinMaxDblAggregator, 
genDeclMethod=System.Void Microsoft.ML.Transforms.Normalizers.MinMaxDblAggregator::Microsoft.ML.Runtime.Data.IColumnAggregator<Microsoft.ML.Runtime.Data.VBuffer<System.Double>>.ProcessValue(Microsoft.ML.Runtime.Data.VBuffer`1<System.Double>& modreq(System.Runtime.InteropServices.InAttribute))
   at Mono.Documentation.Updater.SlashDocMemberFormatter.AppendTypeName(StringBuilder buf, TypeReference type, DynamicParserContext context)
   at Mono.Documentation.Updater.MemberFormatter.AppendGenericType(StringBuilder buf, TypeReference type, DynamicParserContext context, Boolean appendGeneric)
   at Mono.Documentation.Updater.SlashDocMemberFormatter.AppendGenericType(StringBuilder buf, TypeReference type, DynamicParserContext context, Boolean appendGeneric)
   at Mono.Documentation.Updater.MemberFormatter.GetTypeName(TypeReference type, DynamicParserContext context, Boolean appendGeneric)
   at Mono.Documentation.Updater.SlashDocMemberFormatter.AppendParameter(StringBuilder buf, IList`1 genArgs, ParameterDefinition parameter)
   at Mono.Documentation.Updater.SlashDocMemberFormatter.AppendParameters(StringBuilder buf, IList`1 genArgs, IList`1 parameters)
   at Mono.Documentation.Updater.SlashDocMemberFormatter.GetMethodDefinitionName(MethodReference method, String name)
```

so this is an error importing content from a docxml file

--- a/mdoc/Mono.Documentation/Updater/Formatters/SlashDocMemberFormatter.cs
+++ b/mdoc/Mono.Documentation/Updater/Formatters/SlashDocMemberFormatter.cs
@@ -54,19 +54,13 @@ namespace Mono.Documentation.Updater
                         }
                     }
                 }
-                if (genDeclType == null && genDeclMethod == null)
+                if ((genDeclType == null && genDeclMethod == null) || buf.Length == l)
                 {
                     // Probably from within an explicitly implemented interface member,
                     // where CSC uses parameter names instead of indices (why?), e.g.
                     // MyList`2.Mono#DocTest#Generic#IFoo{A}#Method``1(`0,``0) instead of
                     // MyList`2.Mono#DocTest#Generic#IFoo{`0}#Method``1(`0,``0).
                     buf.Append (type.Name);
-                }
-                if (buf.Length == l)
-                {
-                    throw new Exception (string.Format (
-                            "Unable to translate generic parameter {0}; genDeclType={1}, genDeclMethod={2}",
-                            type.Name, genDeclType, genDeclMethod));
                 }
             }
             else

